token/js/src/extensions/transferHook/instructions.ts

```diff
@@ -1,11 +1,14 @@
 import { struct, u8 } from '@solana/buffer-layout';
-import type { PublicKey, Signer } from '@solana/web3.js';
+import type { Commitment, Connection, PublicKey, Signer } from '@solana/web3.js';
 import { TransactionInstruction } from '@solana/web3.js';
 import { programSupportsExtensions, TOKEN_2022_PROGRAM_ID } from '../../constants.js';
 import { TokenUnsupportedInstructionError } from '../../errors.js';
 import { addSigners } from '../../instructions/internal.js';
 import { TokenInstruction } from '../../instructions/types.js';
 import { publicKey } from '@solana/buffer-layout-utils';
+import { createTransferCheckedInstruction } from '../../instructions/index.js';
```

Comment:
nit: can you import this from the actual file for consistency with everything else?
```suggestion
import { createTransferCheckedInstruction } from '../../instructions/transferChecked.js';
```